fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -148,6 +162,15 @@ def self.available_options
                                        verify_block: proc do |value|
                                          UI.user_error!("No API token for Crashlytics given, pass using `api_token: 'token'`") if value.to_s.length == 0
                                        end),
+          FastlaneCore::ConfigItem.new(key: :gsp_path,
+                                      env_name: "GOOGLE_SERVICES_INFO_PLIST_PATH",
+                                      sensitive: true,
```

Comment:
I think this would be better served as `code_gen_sensitive` instead of `sensitive`. Sensitive is usually used for API keys or passwords so that if we prompt for it, you'll get `****` instead of plain text. `code_gen_sensitive` prevents our code generation from pulling in this value when we generate the Swift API.

Reply:
Overlooked that one 🙈 Changed it and fixed the indentation 🙂

Reply:
🎉